adserver/apps.py

```diff
@@ -8,3 +8,4 @@ class AdserverConfig(AppConfig):
 
     def ready(self):
         import adserver.tasks  # noqa
+        import adserver.hooks  # noqa
```

Comment:
I wonder about calling this `signals` instead? It seems that it's using the same pattern, and would be where folks might look for similar logic in a Django codebase. I don't feel strongly, but just a thought.

Reply:
This isn't a signal. This is a django-stripe specific hook. I realize this isn't the standard place but there isn't exactly a standard place to put it. I think `webhooks.py` is better than `signals.py` overall. Since it isn't precisely a true webhook either, I chose `hooks.py`.